src/ploneintranet/workspace/browser/tiles/templates/sidebar.pt

```diff
@@ -258,35 +257,94 @@ Supported types include (class names):
+                                             checked python: 'checked' if task['checked'] else ''"
+                             />
+                      <a class="pat-inject"
+                         tal:attributes="title task/title;
+                                         href python: task['url']+'#document-body'"
+                         tal:content="task/title"/>
+                    </label>
+                  </tal:task>
+                </fieldset>
+              </tal:workspace_tasks>
+
+              <tal:case_tasks condition="ws/is_case"
+                              define="milestones ws_view/metromap_sequence | nothing;
+                                      token context/@@authenticator/token;">
+                <tal:milestone repeat="milestone milestones">
+
+                  <fieldset class="object-list tasks pat-checklist pat-collapsible closed" data-pat-collapsible="store: local"
+                            tal:attributes="id string:milestone-${milestone}">
+                    <h4 class="section-label state-finished" tal:content="python:milestones[milestone]['title']"/>
+                    <tal:task repeat="task python:tasks[milestone]">
+                      <label>
+                        <input type="hidden" name="current-tasks:list" tal:attributes="value task/id"/>
+                        <input type="checkbox" name="active-tasks:list"
+                               tal:attributes="value task/id;
+                                               checked python:'checked' if task['checked'] else ''"
+                               />
+
+                        <a class="pat-inject"
+                           tal:attributes="title task/title; href string:${task/url}#document-body"
+                           tal:content="task/title"/>
+                      </label>
+                    </tal:task>
+                    <p class="button-bar">
+                      <a class="icon-plus-circle small button pat-modal" data-pat-modal="class: large" data-pat-inject="source:#content"
+                         tal:attributes="href string:${ws/absolute_url}/add_task?milestone=${milestone}"
+                         tal:condition="view/can_add">Create new task</a>
+                      <a class="small icon-stage success button"
+                         tal:condition="python:milestones[milestone]['enabled']"
+                         tal:attributes="href python:'{0}/content_status_modify?workflow_action={1}&_authenticator={2}'.format(ws.absolute_url(), milestones[milestone]['transition_id'], token);">Close milestone</a>
+                      <a class="small icon-stage success button"
+                         tal:condition="python:milestones[milestone]['reopen_transition']"
+                         tal:attributes="href python:'{0}/content_status_modify?workflow_action={1}&_authenticator={2}'.format(ws.absolute_url(), milestones[milestone]['reopen_transition'], token);">Reopen milestone</a>
+                    </p>
```

Comment:
These actions should not be GET requests. Plone is a bad example in this case(!)